subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/ticker.h

```diff
@@ -32,16 +38,30 @@
 		/ 1000UL \
 	)
 
+/* Macro to remove ticks and return positive remainder value in microseconds */
+#define HAL_TICKER_REMOVE_JITTER(t, r) \
+	{ \
+		if ((!(r / 1000000UL)) || (r & BIT(31))) { \
+			t--; \
+			r += 30517578UL; \
```

Comment:
Please change a magic number into named constant. The same value is repeated in multiple places hence it is a great candidate to be named constant.